dags/resources/stages/parse/table_definitions/consensys/metamask_swaps_v1_event_AdapterRemoved.json

```diff
@@ -0,0 +1,32 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "string",
+                    "name": "aggregatorId",
+                    "type": "string"
+                }
+            ],
+            "name": "AdapterRemoved",
+            "type": "event"
+        },
+        "contract_address": "0x881d40237659c251811cec9c364ef91dc08d300c",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "consensys",
+        "schema": [
+            {
+                "description": "",
+                "name": "aggregatorId",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "metamask_swaps_v1_event_AdapterRemoved"
```

Comment:
For the `table_name`, we usually camel case this (ie, in this case  `Metamask_Swaps_V1_event_AdapterRemoved`).

Reply:
Ah, no I can change this, thanks.